apps/explorer/lib/explorer/repo/config_helper.ex

```diff
@@ -20,7 +20,7 @@ defmodule Explorer.Repo.ConfigHelper do
   @postgrex_env_vars [
     username: "PGUSER",
     password: "PGPASSWORD",
-    host: "PGHOST",
+    hostname: "PGHOST",
```

Comment:
can we just change this key? have we asserted that nothing in the app  is looking for `:host`?

Reply:
the change is just to match the names in https://github.com/celo-org/blockscout/blob/master/apps/explorer/lib/explorer/repo/config_helper.ex#L42

Reply:
yep, that should be hostname https://hexdocs.pm/ecto_sql/Ecto.Adapters.Postgres.html#module-connection-options

awesome catch, and you are amazing